mobile_verifier/src/service_provider/reward.rs

```diff
@@ -74,28 +56,35 @@ impl ServiceProviderRewardInfos {
     pub fn new(
         dc_sessions: ServiceProviderDCSessions,
         promotions: ServiceProviderPromotions,
-        total_sp_allocation: Decimal,
-        mobile_bone_price: Decimal,
+        total_sp_allocation: Decimal, // Bones
+        mobile_bone_price: Decimal,   // Price in Bones
```

Comment:
i didn't think fmt would let you do a line-ending comment like this but great news if it does